Fix:

error: implicitly declaring library function 'printf' with type 'int (const char *, ...)' [-Werror,-Wimplicit-function-declaration]

https://github.com/gobby/libinfinity/pull/32
--- configure.orig	2020-12-11 15:11:01.000000000 -0600
+++ configure	2023-08-01 15:11:37.000000000 -0500
@@ -14099,6 +14099,7 @@
 $as_echo_n "checking for O_NOFOLLOW... " >&6; }
 cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
+#include <stdio.h>
 #include <sys/types.h>
                 #include <sys/stat.h>
                 #include <fcntl.h>
